app/routes/_layout+/bookings.new.tsx

```diff
@@ -204,6 +223,14 @@ export default function NewBooking() {
               : undefined
           }
         />
+        {assetIds.map((item, i) => (
```

Comment:
This is not the correct place to add this. It should be in the form so it gets sent with the form data. This will never work.

Reply:
Added in the form component